plugins/explore-backend/README.md

```diff
@@ -139,6 +139,21 @@ To index explore tools you will need to register the search collator in the
 ...
 ```
 
+### New Backend System
+
+The explore backend plugin has support for the [new backend system](https://backstage.io/docs/backend-system/), here's how you can set that up:
+
+In your `packages/backend/src/index.ts` make the following changes:
+
+```diff
+  import { createBackend } from '@backstage/backend-defaults';
++ import { explorePlugin } from '@backstage/plugin-explore-backend';
+  const backend = createBackend();
+  // ... other feature additions
++ backend.add(explorePlugin());
```

Comment:
```suggestion
+ backend.add(import('@backstage/plugin-explore-backend'));
```

You can actually remove the import line above, and do this instead (after having changed to a default export as mentioned in the other comment).

Reply:
@freben The readMe file has been updated to reflect the suggestion

Reply:
i think we can just keep the second method honestly :) it's what everybody is expected to want to do going forward.